code/platformio.ini

```diff
@@ -65,6 +65,7 @@ build_flags_4m3m = ${common.build_flags} -Wl,-Teagle.flash.4m3m4s.ld
 # ------------------------------------------------------------------------------
 upload_port = "${sysenv.ESPURNA_IP}"
 upload_flags = --auth="${sysenv.ESPURNA_AUTH}"
+upload_protocol = espota
```

Comment:
Have not really followed the update_protocol syntax. Do you use this in a custom env: ?